Jenkinsfile

```
@@ -1,2 +1,4 @@
-buildPlugin()
-
+buildPlugin(configurations: [
+    [platform: 'linux', jdk: '8'],
+    [platform: 'linux', jdk: '11'],
```

Otherwise you're removing the windows build.
```suggestion
    [platform: 'linux', jdk: '11'],
    [platform: 'windows', jdk: '8'],
```

---

```suggestion
    [platform: 'linux', jdk: '11'],
    [platform: 'windows', jdk: '11'],
```

---

I prefer to go to jdk11 on windows